### ISSUE TITLE
Missing assets in v.0.10.2

### ISSUE THREAD

**Issue by osorochich** [fastlane/snapshot#301]
Hi guys,

First of all, thank you a lot for that amazing tool. I believe this tool can make our QA-team work easier and faster :)

But I'd like to discuss one point regarding missing assets. I know there are very similar problems([#249](https://github.com/fastlane/snapshot/issues/249), [#296](https://github.com/fastlane/snapshot/issues/296)), but I found out very tricky workournd and hope you can find better solution based on this.

So, when I type `snapshot` in command line it builds my project and after that runs my tests. But it looks like some of my assets are missing, but when I build my project under xCode - it works fine and I can see all my assets.

So I did some experiments:

First approach:
1. I remove the app from the simulator
2. I install app that has been built by snapshot on the simulator using command - `xcrun simctl install booted  /tmp/snapshot/build/IHGBranded-Priorityclubrewards-Universal.app`
3. As a result I don't see assets again

Second approach:
1. I remove the app from the simulator
2. I install app that has been build by xCode on the sumulator usuing command - `xcrun simctl install booted  /app_derived_data_path/path_to_dot_app_file`
3. As a result I can see all my assets and everything works fine

So, summing up, I don't know what causes that problem, but maybe it is connected with missing flags or something else. Could you guys help we to solve this problem?

P.S. I use `v0.10.2`, because at the moment `xCode UI Testing` haven't opportunity to scroll to the particular cell on UITableView, but it is critial to me. Hope it will be fixed in future, but for the moment I need to use UI Automation.

**Comment by JesseCrocker** [comment]
I ran into what im guessing is the same issue, and eventually came up with a workaround for it: In the project build settings, under `Asset Catalog Compiler - Options` set `Enable Building Only Active Resources` to No.

Whats causing it:
It's an issue compiling binary asset catalogs that is causing only 1x assets to be included. I've got 2 build machines, one running 10.10 and one running 10.11, both with the latest xcode, and this issue only seems to happen on the 10.11 machine, but there could be some other difference between the 2 and the OS version is not really triggering this.

Here's the warning from xctool:
```
2015-11-09 07:32:35.110 Interface Builder Cocoa Touch Tool[23555:319325] CoreUI: CUICatalog: couldn't get 'kMGArtworkGraphicsFeatureSetClass' from MGCopyAnswer got '{
    ArtworkDeviceIdiom = pad;
    ArtworkDeviceScaleFactor = "-1";
    DevicePerformanceMemoryClass = 1;
}'
2015-11-09 07:32:35.111 Interface Builder Cocoa Touch Tool[23555:319325] CoreUI: CUICatalog: couldn't get 'kMGArtworkGraphicsFeatureSetFallbacks' from MGCopyAnswer got '{
    ArtworkDeviceIdiom = pad;
    ArtworkDeviceScaleFactor = "-1";
    DevicePerformanceMemoryClass = 1;
}'
```

**Comment by osorochich** [comment]
Setting `Enable Building Only Active Resources` to `No` solved my problem. @JesseCrocker thank you a lot!
